### ISSUE TITLE
Rate limit on errors?

### ISSUE THREAD

**Issue by cmnjcs** [meteorhacks/kadira#183]
Is there a rate limit for error tracking? I'm having an issue where code before and after the `Kadira.trackError()` call is running, but Kadira doesn't show any record of that error.

**Comment by arunoda** [comment]
Yep. There are some ratelimits for the same error. We only send a same
error once.

**Comment by cmnjcs** [comment]
This error isn't happening on our end very often (once per hour, give or take), so that doesn't seem to be the issue. Is there a rate limit on the total number of errors (of all types) sent over a given duration from one instance?
